subgraphs/uniswap-forks/config/spookyswap/fantom/fantom.ts

```diff
@@ -0,0 +1,81 @@
+import { Address, BigDecimal } from '@graphprotocol/graph-ts';
+import { Factory } from '../../../generated/Factory/Factory';
+import { FeeSwitch, Network, RewardIntervalType } from '../../../src/common/constants';
+import { Configurations } from '../../configurations/interface';
+
+export class SpookyswapFantomConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.FANTOM;
+  }
+  getProtocolName(): string {
+    return "Spookyswap";
+  }
+  getProtocolSlug(): string {
+    return "spookyswap";
+  }
+  getFactoryAddress(): string {
+    return "0x152ee697f2e276fa89e96742e9bb9ab1f2e61be3";
+  }
+  getFactoryContract(): Factory { 
+    return Factory.bind(Address.fromString("0x152ee697f2e276fa89e96742e9bb9ab1f2e61be3"));
+  }
+  getTradeFee(): BigDecimal {
```

Comment:
I am unsure how to handle fees. It looks like there is a .22% fee for limit orders. Have you seen or heard anything else regarding this?

Reply:
I think we can ignore limit orders. @ken-m-1